### ISSUE TITLE
various for PMID:25987607 

### ISSUE THREAD

**Issue by ValWood** [pombase/fypo#2269]
decreased protein localization to mitotic spindle pole body during M-phase
abolished protein localization to nucleus during mitotic M-phase
abolished protein localization to nucleus during mitotic interphase
abolished protein localization to mitotic spindle during M-phase

standard defs

**Comment by ValWood** [comment]
also 
abnormal protein localization to microtubule minus end

**Comment by mah11** [comment]
new terms
  abolished protein localization to nucleus during mitotic interphase FYPO:0004753
  abolished protein localization to mitotic spindle pole body during mitosis FYPO:0004754
  abolished protein localization to mitotic spindle during mitosis FYPO:0004754
  abnormal protein localization to microtubule minus-end FYPO:0004755

For the first request, I added another synonym to FYPO:0002822 'decreased protein localization to mitotic spindle pole body during mitosis' (it already had the related synonym 'decreased protein localization to mitotic spindle pole body during mitotic M phase').

Similarly, for the second one I added synonyms to FYPO:0003192 'abolished protein localization to nucleus during mitosis', and named the new terms using 'mitosis'.

I've also standardized the other terms  on 'mitosis' in name & xp, and 'mitotic M phase' in related synonyms, because we had a lot more terms with 'during mitosis' to start with, and a few had the 'M phase' synonyms. I can separate them out if it turns out we need to make a distinction (would need an explanation of what the distinction is, for the text defs).

... and I included FYPO:0004754 by mistake because I got my head spun right round, but it won't hurt to keep it in case it's useful later.
